src/super_gradients/training/models/classification_models/mobilenetv2.py

```diff
@@ -128,20 +128,17 @@ def __init__(
             output_channel = make_divisible(c * width_mult) if t > 1 else c
             for i in range(n):
                 if i == 0:
-                    self.features.append(
-                        block(curr_channels, output_channel, s, expand_ratio=t, grouped_conv_size=grouped_conv_size)
-                    )
+                    self.features.append(block(curr_channels, output_channel, s, expand_ratio=t, grouped_conv_size=grouped_conv_size))
                 else:
-                    self.features.append(
-                        block(curr_channels, output_channel, 1, expand_ratio=t, grouped_conv_size=grouped_conv_size)
-                    )
+                    self.features.append(block(curr_channels, output_channel, 1, expand_ratio=t, grouped_conv_size=grouped_conv_size))
                 curr_channels = output_channel
         # building last several layers
         self.features.append(conv_1x1_bn(curr_channels, self.last_channel))
         # make it nn.Sequential
         self.features = nn.Sequential(*self.features)
+        self.backbone_mode = backbone_mode
 
-        if backbone_mode:
+        if self.backbone_mode:
```

Comment:
Why do we still need self.backbone_connection_channels attribute with the new design?

Reply:
In the new design, no, but I'm not sure about the old one, afraid to break something. Let's get back to this when we refactor yolos

Reply:
Please add a @todo comment so wed remember to do so (hopefully).